### DISCUSSION TITLE
My latest release are not marked as latest

### DISCUSSION THREAD

**Discussion by baris-inandi** [community/community#22198] [New to GitHub]
<p>I have 4 releases in my repo and I am trying to access the tag of the latest release through the github api. I looked it up on the support community <a href="https://github.community/t/why-last-release-is-not-marked-as-latest-release/421">here</a>, all of the releases are created via the github web interface and there are exactly 4 tags. It seems that this might not be happening becuse of the [quote=“cmrberry, post:2, topic:421”]<br>
everything you see there are tags that were pushed to the repository without having a release formally created.<br>
[/quote]<br>
issue, why does my repo don’t have a “latest release” ?<br>
<a href="https://api.github.com/repos/baris-inandi/fileencryptor/releases/latest" rel="noopener nofollow ugc">https://api.github.com/repos/baris-inandi/fileencryptor/releases/latest</a> is not fouınd</p>

**Comment by killianeletellier** [comment, marked as answer]
<p>Hello,<br>
I think it’s beacause all of your releases are “pre-releases” and no “releases”.<br>
Submit a release which you don’t mark as a pre-release and try to get your latest release via the API.</p>
<p>LProgead.</p>

**Comment by baris-inandi** [comment]
<p>Was that all? I marked it as a full release and that did it!<br>
Thanks :smiley:</p>

**Comment by killianeletellier** [comment]
<p>Nice ! Don’t forget to mark it as a solution :wink:</p>
<p>LProgead.</p>
